analytic_estimated_amount/models/purchase_order_line.py

```diff
@@ -28,7 +34,7 @@ def _generate_analytic_line(self):
         self._remove_analytic_lines()
         line_cls = self.env['account.analytic.line']
         for line in self:
-            if line.account_analytic_id:
+            if line._estimatitive_line_required():
```

Comment:
estimatitive -> estimative

I don't care that much about a typo here and there but since the method name is the API for the 'hook' in this case it should be corrected